src/hooks/useStore.js

```diff
@@ -1,16 +1,32 @@
 import create from 'zustand';
 
-const useStore = create(set => ({
-	counter: 0,
-	setCounter(counter) {
-		set({counter});
+export const useStore = create(set => ({
+	arrData: [],
+	arrConfiguration: [],
+	isConfigurationLoaded: false,
+
+	setData: item => {
+		// eslint-disable-next-line no-unused-vars
```

Comment:
Do you really need this here? And if yes, why? 🤔

Reply:
This is "one for one" the example we used, works fine, but now the linter says he doesn't like the state in line 10. If i add the state to the next line like arrData: state.item instead of (arrData: item), he says that item (line 8) isn't used.

Reply:
I will have a look at that 🙂

Reply:
Maybe try the following:

```
set(() => {
			return {
				arrData: item,
			};
		});
```

Reply:
Did you test Andreas suggestion? 🧐